web/oauth_test.go

```diff
@@ -617,35 +563,27 @@ func GenerateTestAppName() string {
 func CheckNoAppError(t *testing.T, err *model.AppError) {
 	t.Helper()
 
-	if err != nil {
-		t.Fatalf("Expected no error, got %q", err.Error())
-	}
+	require.Nil(t, err, "Expected no error, got %q", err.Error())
```

Comment:
This is causing an NPE. Would you please just remove the function in whole and just use `require.Nil`?

Reply:
Hey @hanzei, thanks for the review.

I tried to follow your suggestion, but the build has failed again. Can you help me understand what could have possibly gone wrong?

Reply:
I looked into but didn't found the cause either. :man_shrugging: